package.json

```diff
@@ -93,6 +93,7 @@
     "jest-haste-map@^29.7.0": "patch:jest-haste-map@npm%3A29.7.0#./.yarn/patches/jest-haste-map-npm-29.7.0-e3be419eff.patch"
   },
   "dependencies": {
+    "@backstage/cli-node": "workspace:^",
```

Comment:
You probably wanted to add this to your package instead of the root.

Reply:
Fixed.